value/src/main/java/com/google/auto/value/processor/AutoValueProcessor.java

```diff
@@ -434,26 +444,24 @@ private void processType(TypeElement type) {
     }
   }
 
-  private AutoValueExtension.Context makeExtensionContext(
-      final TypeElement type, final Iterable<ExecutableElement> methods) {
-    return new AutoValueExtension.Context() {
-
-      @Override public ProcessingEnvironment processingEnvironment() {
-        return processingEnv;
-      }
-
-      @Override public String packageName() {
-        return TypeSimplifier.packageNameOf(type);
-      }
-
-      @Override public TypeElement autoValueClass() {
-        return type;
-      }
+  private <T> ImmutableSet<T> newFilteredImmutableSet(ImmutableSet<T> original,
+                                                      Predicate<T> predicate) {
+    ImmutableSet.Builder<T> builder = ImmutableSet.builder();
+    for (T item : original) {
+      if (predicate.apply(item)) builder.add(item);
+    }
+    return builder.build();
+  }
 
-      @Override public Map<String, ExecutableElement> properties() {
-        return propertyNameToMethodMap(methods);
+  private <K, V> ImmutableBiMap<K, V> newImmutableBiMapRemovingKeys(ImmutableBiMap<K, V> original,
```

Comment:
I almost made the same mistake with `newFilteredImmutableSet` and `Sets.filter`...